### ISSUE TITLE
Service name  inference by port in kat_nmap normalizer

### ISSUE THREAD

**Issue by originalsouth** [minvws/nl-kat-coordination#2037]
Question regarding the following lines of code: 
https://github.com/minvws/nl-kat-coordination/blob/56c8004ba41e9f6d3d7ba94920048e361278a78a/boefjes/boefjes/plugins/kat_nmap/normalize.py#L46-L50

Suppose one places a `ssh` server behind port 80 or port 443, do we really want to report `http` or `https` respectively?
Perhaps there are considerations for this behavior?

**Comment by noamblitz** [comment]
No, then it should not be overwritten bij http or https. I remember a thing where nmap would identify services like http-web instead of http. Maybe this was implemented to overwrite those...?

**Comment by originalsouth** [comment]
> No, then it should not be overwritten bij http or https. I remember a thing where nmap would identify services like http-web instead of http. Maybe this was implemented to overwrite those...?

Could very well be, nmap reports the following http type service names:
```
http
http-ocsp
http-proxy
http-proxy-ctrl
kazaa-http
ncacn_http
ntop-http
ssl/http
vnc-http
```

We could opt for a simplified re-classification where suited?
For instance:
```
http           -> http
http-proxy     -> http
ssl/http       -> https
```

**Comment by noamblitz** [comment]
Sounds better to me

**Comment by originalsouth** [comment]
Addressed in 78a830e.

It is better to use tunnel type in stead of service name as that is the format used in the XML output (rather than the standard output).
